<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="EmpPhoto">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>EmpPhoto</string>
   </property>
  </widget>
  <widget class="QLabel" name="EmpName">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>254</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>EmpName</string>
   </property>
  </widget>
  <widget class="QLabel" name="EmpID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>EmpID</string>
   </property>
  </widget>
  <widget class="QLabel" name="EmpDep">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>EmpDep</string>
   </property>
  </widget>
  <widget class="QLabel" name="EmpJobTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>EmpJobTitle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
